app/integration/ae-integration/src/main/java/uk/ac/ebi/fg/annotare2/integration/AeIntegrationWatchdog.java

```diff
@@ -363,18 +363,42 @@ public void processSubmitted(Submission submission) throws SubsTrackingException
 
     private void moveExportDirectory(File exportDir) throws SubsTrackingException {
         LinuxShellCommandExecutor executor = new LinuxShellCommandExecutor();
-        String lsfToDatamoverScript = properties.getLSFtoDataMoverScript() + " \"" + exportDir.getPath() + "\"";
+        String moveExportDirectoryScript = properties.getMoveExportDirectoryScript() + " \"" + exportDir.getPath() + "\"";
         try {
-            logger.info("Moving export directory {} to codon started.", exportDir.getPath());
-            executor.execute("ssh codon-login \"bash -s\" < " + lsfToDatamoverScript);
-            logger.info("Moving export directory {} to codon finished.", exportDir.getPath());
-            executor.execute("rm -rf " + exportDir.getPath());
-            logger.info("Export directory {} deleted.", exportDir.getPath());
+            String jobId = submitMoveExportDirectorySlurmJob(exportDir, executor, moveExportDirectoryScript);
+            if(getSlurmJobStatus(jobId, executor).equals("COMPLETED")){
+                deleteExportDirectory(exportDir, executor);
+            }
         } catch (IOException e) {
             throw new SubsTrackingException(e);
         }
     }
 
+    private static void deleteExportDirectory(File exportDir, LinuxShellCommandExecutor executor) throws IOException {
+        executor.execute("rm -rf " + exportDir.getPath());
+        logger.info("Export directory {} deleted.", exportDir.getPath());
+    }
+
+    private String submitMoveExportDirectorySlurmJob(File exportDir, LinuxShellCommandExecutor executor, String moveExportDirectoryScript) throws IOException {
+        logger.info("Moving export directory {} to codon started.", exportDir.getPath());
+        executor.execute("ssh codon-slurm-login \"bash -s\" < " + moveExportDirectoryScript);
+        logger.info("Moving export directory {} to codon finished.", exportDir.getPath());
+        return !isNullOrEmpty(executor.getOutput()) ? executor.getOutput().replaceAll("\\D+", "") : null;
+    }
+
+    private String getSlurmJobStatus(String jobId, LinuxShellCommandExecutor executor) throws IOException {
+        executor.execute("ssh codon-slurm-login jobinfo " + jobId);
+        String[] lines = executor.getOutput().split("\n");
+        String state = null;
+        for (String line : lines) {
+            if (line.startsWith("State")) {
+                state = line.split(":")[1].trim();
+                break;
+            }
+        }
+        return state;
+    }
```

Comment:
Here is the modified method for the above comment.

```suggestion
    private boolean isSlurmJobFinished(String jobId, LinuxShellCommandExecutor executor) throws IOException {
        executor.execute("ssh codon-slurm-login jobinfo " + jobId);
        String[] lines = executor.getOutput().split("\n");
        
        return Arrays.stream(lines).anyMatch(text -> text.contains("COMPLETED"));        
    }
```